### DISCUSSION TITLE
Information message on homepage

### DISCUSSION THREAD

**Discussion by Djf2884** [netbox-community/netbox#8379] [Q&A] [answered]
Hello Guys, 

I'd like to know if there is a possibility to show an information message on the homepage for user that login? or for all users.

Thx you :)

**Comment by kkthxbye-code** [comment, marked as answer]
The following settings can be used to inject content:

BANNER_LOGIN
BANNER_TOP
BANNER_BOTTOM

https://netbox.readthedocs.io/en/stable/configuration/dynamic-settings/

**Comment by Djf2884** [comment]
Thx exactly what i was looking for :)
